tenseal/tensors/bfvvector.cpp

```diff
@@ -171,6 +171,11 @@ BFVVector& BFVVector::mul_inplace(BFVVector to_mul) {
 
     this->context->evaluator->multiply_inplace(this->ciphertext,
                                                to_mul.ciphertext);
+                                               
+    // TODO: make this optional
+    // relineraization after ciphertext-ciphertext multiplication
+    this->context->evaluator->relinearize_inplace(this->ciphertext,
+                                                  this->context->relin_keys());
```

Comment:
Wouldn't be useful to support relinearization after a batch of operations? 
Meaning:
- you could let the user choose when to relinearize.
 - or you could let them do a set of operations in a row that have at least a ct-ct multiplication, and just after them to relinearize.

Reply:
We aim at providing the ability to the user to disable relinearization in the near future so that he can do that himself, but the main rule for the moment is to relinearize after each mul to keep the ciphertext at its smallest size